src/grid.ts

```diff
@@ -41,10 +41,49 @@ export const drawGrid = (x, y, gridX, gridY, props, ticks) => {
   }
 
   if (grid.y.visible) {
-    // add the Y gridlines
+    // add the Y grid lines
+    gridY.attr('transform', `translate(${offset.x}, 0)`)
+      .transition()
+      .call(makeYGridLines(y, get(grid, 'y.ticks', ticks))
+        .tickSize(-width + (margin.left * 2) + yAxisWidth(axis))
+        .tickFormat(() => ''),
+      );
+
+    attrs(gridY.selectAll('.tick line'), grid.y.style);
+
+    // Hide the first horizontal grid line to show axis
+    gridY.selectAll('.gridY .tick line').filter((d, i) => i === 0)
+      .attr('display', 'none');
+    attrs(gridY.selectAll('.domain'), { ...axis.x.style, stroke: 'transparent' });
+  }
+};
+
+export const drawHorizontalGrid = (x, y, gridX, gridY, props, ticks) => {
```

Comment:
there are no types on these, would probably be quite useful particularly for the `props`

Reply:
also it seems like almost all of these arguments are unused

Reply:
I've updated the props - think they are all used though